web/src/components/dashboard/anytime-actions/AnytimeActionDropdown.tsx

```diff
@@ -41,10 +41,9 @@ export const AnytimeActionDropdown = (props: Props): ReactElement => {
 
   const licenseReinstatementMatch = (action: AnytimeActionLicenseReinstatement): boolean => {
     return !!(
-      action.industryIds &&
-      industryId &&
-      action.industryIds.includes(industryId) &&
-      business.licenseData?.status === "EXPIRED"
+      (action.industryIds && industryId && action.industryIds.includes(industryId))
+      // &&
+      // business.licenseData?.status === "EXPIRED"
```

Comment:
We don't require the license to be expired to be eligible for reinstatement anymore?